docs/news.rst

```diff
@@ -3,6 +3,59 @@
 Release notes
 =============
 
+1.2.0 (YYYY-MM-DD)
+------------------
+
+New Features
+~~~~~~~~~~~~
+
+- New :setting:`FEED_EXPORT_ENCODING` setting to customize the encoding
+  used when writing items to a file. This is useful for those wanting
+  something else than UTF-8 for XML or CSV output (:issue:`2034`).
```

Comment:
yup, with this now people can have their JSON in UTF-8 (no need to ensure_ascii).